### ISSUE TITLE
2.9.10 patch release plan

### ISSUE THREAD

**Issue by joeparsons** [az-digital/az_quickstart#3436]
- #3403 

## Pick list
```
git cherry-pick 718c76724c612c1953ba31467f93560c326a0482
```

**Comment by trackleft** [comment]
LGTM

**Comment by joeparsons** [comment]
## Release notes snippet
`git log 2.9.9..HEAD --pretty='format:- %s'`
```
- Closes #3434: Update drupal/asset_injector requirement from 2.19.0 to 2.20.0 (#3403)
- Back to dev.
```

**Comment by joeparsons** [comment]
https://github.com/az-digital/az_quickstart/releases/tag/2.9.10
